runtime/src/lib.rs

```diff
@@ -1590,6 +1591,76 @@ impl pallet_crowdloan::Config for Runtime {
     type MaxContributors = MaxContributors;
 }
 
+fn contracts_schedule<T: pallet_contracts::Config>() -> pallet_contracts::Schedule<T> {
+    pallet_contracts::Schedule {
+        limits: pallet_contracts::Limits {
+            runtime_memory: 1024 * 1024 * 1024,
+            validator_runtime_memory: 1024 * 1024 * 1024 * 2,
+            ..Default::default()
+        },
+        ..Default::default()
+    }
+}
+
+parameter_types! {
+    pub const ContractDepositPerItem: Balance = deposit(1, 0);
+    pub const ContractDepositPerByte: Balance = deposit(0, 1);
+    pub const ContractDefaultDepositLimit: Balance = deposit(1024, 1024 * 1024);
+    pub ContractsSchedule: pallet_contracts::Schedule<Runtime> = contracts_schedule::<Runtime>();
+    pub const CodeHashLockupDepositPercent: Perbill = Perbill::from_percent(0);
```

Comment:
This needs to be non-zero per docs: https://docs.rs/pallet-contracts/latest/pallet_contracts/pallet/trait.Config.html#associatedtype.CodeHashLockupDepositPercent

This example has it at 30pc
https://github.com/paritytech/substrate/blob/033d4e86cc7eff0066cd376b9375f815761d653c/bin/node/runtime/src/lib.rs#L1292